packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -15,7 +15,7 @@ import type {StyleType} from "@khanacademy/wonder-blocks-core";
 // NOTE(jeff): Here we share some code for use with PopperJS. Long term,
 // we should either contribute this code to the PopperJS component, or its
 // own non-wonder-blocks package.
-// eslint-disable-next-line import/no-restricted-paths
+// eslint-disable-next-line import/no-restricted-paths $FlowIgnoreMe
```

Comment:
lol, apparently this causes eslint to not work - I'm not sure how to cause both to apply to the next line!